includes/class-wc-gateway-mondido-hw.php

```diff
@@ -269,10 +257,56 @@ public function process_payment( $order_id ) {
 			}
 		}
 
-		return array(
-			'result'   => 'success',
-			'redirect' => $order->get_checkout_payment_url( TRUE )
-		);
+		$transaction_id = $order->get_transaction_id();
+
+		if ($transaction_id) {
+			$transaction = $this->transaction->get($transaction_id);
+
+			if (!is_wp_error($transaction)) {
+				$transaction = $this->transaction->update(
+					$transaction->id,
+					$order,
+					$this->merchant_id,
+					$this->testmode === 'yes',
```

Comment:
Always testmode?

Reply:
its only true if testmode has value of "yes"